ch-covidcertificate-backend-verifier/ch-covidcertificate-backend-verifier-data/src/main/java/ch/admin/bag/covidcertificate/backend/verifier/data/impl/JdbcVerifierDataServiceImpl.java

```diff
@@ -125,46 +135,67 @@ public void insertManualDsc(DbDsc dsc) {
     @Override
     @Transactional
     public int removeDscsNotIn(List<String> keyIdsToKeep) {
-        var sql = "delete from t_document_signer_certificate";
-        final var params = new MapSqlParameterSource();
         if (!keyIdsToKeep.isEmpty()) {
```

Comment:
is this check still needed? in my understanding, if an empty list is passed to this method, all certificates (except for the manually added) should be flagged as deleted.